code/game/objects/items/granters.dm

```diff
@@ -789,114 +789,86 @@
 	crafting_recipe_types = list(/datum/crafting_recipe/scoutcarbine)
 
 /obj/item/book/granter/trait/forgemaster
-	name = "The secrets of the Legion Forgemasters"
-	desc = "All you need to bend the steel right."
+	name = "Ashes of Phoenix: Imperial Weaponsmithing"
```

Comment:
```suggestion
	name = "The secrets of the Legion Forgemasters"
```

Please leave the title as before, rest is fine.

Reply:
The old title also isn't good, a new title is needed I believe. Have any suggestions on what it should be?

Reply:
Something simple is enough, its a type of book thats just more or less placeholder due to limitations in our current loadout code. "Forgemaster manual" maybe. Not worth too much effort or flourish, simple descriptive is fine for that style of loadout book, hopefully we can make them truly obsolete soon.